src/components/AssetsUploadConfirm/container.jsx

```diff
@@ -0,0 +1,22 @@
+import { connect } from 'react-redux';
+
+import { uploadAssets, clearPreUploadProps } from '../../data/actions/assets';
+import AssetsUploadConfirm from '.';
+
+const mapStateToProps = state => ({
+  files: state.metadata.files,
```

Comment:
@mavidser I am still seeing the Proptype error for the files.
<img width="856" alt="Screenshot 2023-10-19 at 3 45 57 PM" src="https://github.com/openedx/studio-frontend/assets/383862/e067a2dd-8f52-421f-9ac8-f554ced98e7d">

Inspecting the component, it can be seen that these are `File` objects and not `string`

<img width="531" alt="Screenshot 2023-10-19 at 3 45 41 PM" src="https://github.com/openedx/studio-frontend/assets/383862/8f7c3b6e-1d7d-4c08-ad2f-5911131c1928">

The proptype for the `files` should be set to `object` in index.js.